src/lib/message.sjs

```diff
@@ -191,6 +191,23 @@ var extensions = {
         message.replyname = "RPL_WHOISBOT";
     },
 
+    "346": function (message) {
+        // :<server> 346 <me> <channel> <banmask> <banner> <bantime>
+        message.replyname = "RPL_INVITELIST";
+        message.channel = message.params[1];
+        message.setmask = message.params[2];
+        message.setnick = message.params[3];
+        message.settime = message.params[4];
```

Comment:
`message.setnick` should be `who`, and `message.settime` should be `timestamp` to align with `RPL_TOPICWHOTIME`.

`message.setmask` should be `.whoHostmask` I think.